forge-game/src/main/java/forge/game/card/Card.java

```diff
@@ -4525,6 +4533,71 @@ public final StaticAbility getStaticAbilityForStaticAbility(final String str, fi
         return result;
     }
 
+    public final SpellAbility getSpellAbilityForStaticAbilityByText(final SpellAbility sa, final StaticAbility stAb) {
+        SpellAbility result = storedSpellAbililityByText.get(stAb, sa);
+        if (result == null) {
+            result = sa.copy(this, false);
+            result.setOriginalAbility(sa); // need to be set to get the Once Per turn Clause correct
+            result.setGrantorStatic(stAb);
+            result.setIntrinsic(true); // needs to be changed by CardTextChanges
+            storedSpellAbililityByText.put(stAb, sa, result);
+        }
+        return result;
+    }
+
+    public final SpellAbility getSpellAbilityForStaticAbilityGainedByText(final String str, final StaticAbility stAb) {
+        SpellAbility result = storedSpellAbililityGainedByText.get(stAb, str);
+        if (result == null) {
+            result = AbilityFactory.getAbility(str, this, stAb);
+            result.setIntrinsic(true); // needs to be affected by Text
+            result.setGrantorStatic(stAb);
+            storedSpellAbililityGainedByText.put(stAb, str, result);
+        }
+        return result;
+    }
+
+    public final Trigger getTriggerForStaticAbilityByText(final Trigger tr, final StaticAbility stAb) {
+        Trigger result = storedTriggerByText.get(stAb, tr);
+        if (result == null) {
+            result = tr.copy(this, false);
+            result.setIntrinsic(false); // needs to be changed by CardTextChanges
+            storedTriggerByText.put(stAb, tr, result);
+        }
+        return result;
+    }
+
+    public final ReplacementEffect getReplacementEffectForStaticAbilityByText(final ReplacementEffect re, final StaticAbility stAb) {
+        ReplacementEffect result = storedReplacementEffectByText.get(stAb, re);
+        if (result == null) {
+            result = re.copy(this, false);
+            result.setIntrinsic(false); // needs to be changed by CardTextChanges
```

Comment:
why are these now set to false? this breaks it but the comment says it should work?